boards/nrf51dk/README.md

```diff
@@ -25,11 +25,21 @@ You can omit `TOCK_BOARD` if you run `make flash` in the nrf51dk directory.
 
 ### Programming user-level applications
 
-After building (`make` in the application folder), you can program an
-application using tockloader:
+You can program an application in two ways:
+ 1. via `tockloader`:
+
+    ```bash
+    $ cd userland/examples/<app>
+    $ make
+    $ tockloader install --jtag --board nrf51dk --arch cortex-m0
+    ```
+
+ 2. Alternatively, via `flash`:
```

Comment:
You might clarify that the `flash` directive to make just wraps that same tockloader command.